include/envoy/grpc/status.h

```diff
@@ -5,9 +5,11 @@ namespace Grpc {
 
 class Status {
 public:
+  using GrpcStatus = int64_t;
+
   // If this enum is changed, then the std::unordered_map in Envoy::Grpc::Utility::nameToGrpcStatus
   // located at: //source/common/access_log/grpc/status.cc must also be changed.
-  enum GrpcStatus {
+  enum GrpcStatusMapping {
```

Comment:
Name this `WellKnownGrpcStatus`? and change `MaximumValid` to `MaximumKnown` or something.